test/kuttl/common/scripts/check_debug_in_neutron_pod_logs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+found=0
```

Comment:
you definitely don't need two flags for the same state. pick one and calculate the other as needed.